MediaBrowser.Model/Configuration/EncodingOptions.cs

```diff
@@ -53,8 +69,19 @@ public EncodingOptions()
             EnableThrottling = false;
             ThrottleDelaySeconds = 180;
             EncodingThreadCount = -1;
-            // This is a DRM device that is almost guaranteed to be there on every intel platform, plus it's the default one in ffmpeg if you don't specify anything
+            // This is a DRM device that is almost guaranteed to be there on every intel platform,
+            // plus it's the default one in ffmpeg if you don't specify anything
             VaapiDevice = "/dev/dri/renderD128";
+            // This is the OpenCL device that is used for tonemapping.
+            // The left side of the dot is the platform number, and the right side is the device number on the platform.
+            OpenclDevice = "0.0";
+            EnableTonemapping = false;
+            TonemappingAlgorithm = "reinhard";
```

Comment:
I personally prefer `hable` or `mobius` to `reinhard` look for most films.

Here were some tests: https://github.com/jellyfin/jellyfin/issues/415#issuecomment-572796507 (and the rest of that thread)
Also the filters seem to really like if the input is scaled to float first. (And I suspect the GPU doesn't really mind)

Reply:
This hardware accelerated `tonemap_opencl` filter is different from `tonemap`. Its average brightness is calculated in real time, which means it will dynamically adjust the brightness according to the `threshold` when the scene brightness changes greatly. I also made some attempts on `hable`, but failed to get a suitable brightness as `reinhard`.

If you also have an nvidia card, you can also try this PR to find better parameters.